mod/metrics/set/abstract/metric_thumbnail.rb

```diff
@@ -22,11 +22,7 @@ def thumbnail_title_without_link
   end
 
   def thumbnail_title_link
-    opts = { title: card.name }
-    if voo.closest_live_option(:project)
-      opts[:path] = { filter: { project: voo.closest_live_option(:project) } }
-    end
```

Comment:
I couldn't find use of this; it appears to have been replaced with the `metric_links` functionality.